winterfell/pdf-filler/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.3/apache-maven-3.3.3-bin.zip
```

Comment:
These files are required for the functionality of the maven wrapper, they are designed to be checked into source control. Mvn wrapper allows you to run a consistent version of maven via mvnw, its more reliable then assuming mvn in installed on the local system